WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -276,10 +275,7 @@ public static void openNoteForReply(Activity activity,
         if (!TextUtils.isEmpty(replyText)) {
             detailIntent.putExtra(NOTE_PREFILLED_REPLY_EXTRA, replyText);
         }
-        if (allowNavigateList) {
-            detailIntent.putExtra(NOTE_ALLOW_NAVIGATE_LIST_EXTRA, true);
```

Comment:
Now that `NOTE_ALLOW_NAVIGATE_LIST_EXTRA` is no longer used, it can be dropped entirely (it's still defined in this fragment).